views/bt_mouse.c

```diff
@@ -100,6 +101,14 @@ static void bt_mouse_process(BtMouse* bt_mouse, InputEvent* event) {
                 } else if(event->type == InputTypeRelease) {
                     bt_mouse_button_state(bt_mouse, HID_MOUSE_BTN_WHEEL, false);
                 }
+            } else if(event->key == InputKeyRight) {
+                if(event->type == InputTypePress) {
```

Comment:
Here and in the remaining three instances, please add ` || event->type == InputTypeRepeat`, otherwise you'd need to tap the button repeatedly to keep scrolling, as opposed to just keeping it pressed.